### DISCUSSION TITLE
Client Library Documentation (Python)

### DISCUSSION THREAD

**Discussion by cobblestone-fsh** [supabase/supabase#2727] [Feature Requests]
The documentation for supabase client libraries shows several Python interactions with the database as not implemented.

https://supabase.io/docs/guides/client-libraries

However, I can mimic the behavior implemented in JavaScript by using what is defined in postgrest-py.

https://github.com/supabase/postgrest-py

I can access the `from_` method from supabase_py like this:

```python
s = supabase_py.create_client(url, key)
s.from_('countries').select('*').limit(10).execute()
```

The documentation could be updated to reflect this (unless this behavior is incomplete).

**Comment by J0** [comment]
Hey!

Thanks for pointing this out! Seems like [the docs](https://supabase.com/docs/guides/client-libraries) have been updated to reflect this. Unfortunately, we are not quite done with `realtime` yet so it hasn't been updated but we will update it once we're done :)


Feel free to let us know if you face any further issues facing the lib.
